MajorNormalization.md

```diff
@@ -44,9 +45,10 @@ Sample Response
             {
                 "normalized_name": "Electrical Engineering",
                 "cip_code": 15.001,
-                "confidence": 1.0,
+                "confidence": 0.8885
             }
-        ]
+        ],
+        "data_version":"1.0.0"
```

Comment:
the other fields have a space after their colons, but this one doesn't.

**LITERALLY UNREADABLE**